pandas/core/indexes/multi.py

```diff
@@ -3502,98 +3502,9 @@ def equal_levels(self, other) -> bool:
     # --------------------------------------------------------------------
     # Set Methods
 
-    def union(self, other, sort=None):
```

Comment:
can you add these examples to the existing Index.union doc-string?

Reply:
restored + greenish